docs/modules/trino/pages/usage-guide/operations/graceful-shutdown.adoc

```diff
@@ -106,19 +111,34 @@ Until native support is added, you will have to use `configOverrides` to enable
 
 == Authorization requirements
 
-WARNING: When you are not using OPA for authorization, the user `admin` is not allowed to gracefully shut down workers.
-If you need graceful shutdown you need to use OPA or need to make sure `admin` is allowed to gracefully shut down workers (e.g. having you own authorizer or patching Trino).
+WARNING: When you are not using OPA for authorization, the user `graceful-shutdown-user` is not allowed to gracefully shut down workers.
+If you need graceful shutdown you need to use OPA or need to make sure `graceful-shutdown-user` is allowed to gracefully shut down workers (e.g. having you own authorizer or patching Trino).
+
+In case you use OPA to authorize Trino requests, you need to make sure the user `graceful-shutdown-user` is authorized to trigger a graceful shutdown of the workers.
 
-In case you use OPA to authorize Trino requests, you need to make sure the user `admin` is authorized to trigger a graceful shutdown of the workers.
-You can achieve this e.g. by adding the following rule, which grants `admin` the permissions to do anything - including graceful shutdown.
+When you are using rego rules provided by Stackable this permission is automatically granted.
+However, when you are writing your own rego rules, you can achieve this e.g. by adding the following rule, which grants `graceful-shutdown-user` the permissions to issue a graceful shutdown.
 
 [source,rego]
 ----
 allow {
-  input.context.identity.user == "admin"
+  input.action.operation == "WriteSystemInformation"
+  input.context.identity.user == "graceful-shutdown-user"
 }
 ----
 
+In Trino policies the following statement is recommended:
+[source,rego]
+----
+"system_information": [
+  # This is needed for graceful shutdown
+  {
+    "user": "graceful-shutdown-user",
+    "allow": ["read", "write"]
+  },
+],
+----
+
 In case the user `admin` does not have the permission to gracefully shut down a worker, the error message `curl: (22) The requested URL returned error: 403 Forbidden` will be shown in the worker log and the worker will shut down immediately.
```

Comment:
```suggestion
In case the user `graceful-shutdown-user` does not have the permission to gracefully shut down a worker, the error message `curl: (22) The requested URL returned error: 403 Forbidden` will be shown in the worker log and the worker will shut down immediately.
```